sql/src/main/java/io/crate/analyze/TableIdentsExtractor.java

```diff
@@ -122,17 +122,17 @@
         public Collection<RelationName> visitMatchPredicate(MatchPredicate matchPredicate, Void context) {
             Set<RelationName> relationNames = new HashSet<>();
             for (Map.Entry<Field, Symbol> entry : matchPredicate.identBoostMap().entrySet()) {
-                relationNames.addAll(process(entry.getKey(), context));
-                relationNames.addAll(process(entry.getValue(), context));
+                relationNames.addAll(((Symbol) entry.getKey()).accept(this, context));
```

Comment:
just wondering why do we need to cast it to `Symbol` here?

Reply:
Probably not required, the inline method refactoring in intellij is sometimes a bit strange. I'll remove it.